File: WIMS-front/src/app/Observables/Currency.ts
export class Currency {

    constructor(private id: number) {}

    private currencyName: string;
    private currencyShortcut: string;
    private conversionRate: number;

    get Id(): number {
        return this.id;
    }

    set Id(value: number) {
        this.id = value;
    }

    get CurrencyName(): string {
        return this.currencyName;
    }

    set CurrencyName(value: string) {
        this.currencyName = value;
    }

    get CurrencyShortcut(): string {
        return this.currencyShortcut;
    }

    set CurrencyShortcut(value: string) {
        this.currencyShortcut = value;
    }

    get ConversionRate(): number {
        return this.conversionRate;
    }

    set ConversionRate(value: number) {
        this.conversionRate = value;
    }

}

File: back_end_fg/WebApplication1/WebApplication1/Models/Currency.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WimsApiMKI.Models
{
    public class Currency
    {
        private int _id;
        private string _currencyName;
        private string _currencyShortcut;
        private decimal _conversionRate;

        public Currency()
        {
            this.Id = 1;
        }

        #region GetSet
        public decimal ConversionRate
        {
            get { return _conversionRate; }
            set { _conversionRate = value; }
        }

        public string CurrencyShortCut
        {
            get { return _currencyShortcut; }
            set { _currencyShortcut = value; }
        }

        public string CurrencyName
        {
            get { return _currencyName; }
            set { _currencyName = value; }
        }

        public int Id
        {
            get { return _id; }
            set { _id = value; }
        }
        #endregion
    }
}
